diesel/src/r2d2.rs

```diff
@@ -1,4 +1,6 @@
-//! Connection pooling via r2d2
+//! Connection pooling via r2d2.
+//! Note that In order to use this module, you need to have Diesel
```

Comment:
"Note that *in* order to use this module you need to enable the `r2d2` feature on diesel." 
sounds better for me.

Reply:
Oops, my bad. Fixed